### PR TITLE
Changed RemoteDebuggingVersion from string type to enum and added tests.

### PR DESCRIPTION
**Pull request by haocs**

### CODE REVIEW

File: src/ServiceManagement/Common/Commands.ScenarioTest/Resources/Websites/WebsitesTests.ps1 [hunk 1]
```diff
@@ -856,6 +857,13 @@ function Test-SetAzureWebsite
     $website = Get-AzureWebsite $name -Slot Production
     Assert-AreEqual Classic $website.ManagedPipelineMode
     Assert-AreEqual $true $website.WebSocketsEnabled
+
+	$website.RemoteDebuggingEnabled = $true
+	$website.RemoteDebuggingVersion = $remotedebuggingversion
+
+	Set-AzureWebsite $name -Slot Production -SiteWithConfig $website
+	Assert-AreEqual $true $website.RemoteDebuggingEnabled
+	Assert-AreEqual $remotedebuggingversion $website.RemoteDebuggingVersion
```
Comment (hovsepm) [inline src/ServiceManagement/Common/Commands.ScenarioTest/Resources/Websites/WebsitesTests.ps1]: please align indentation.

File: src/ServiceManagement/Common/Commands.ScenarioTest/Resources/Websites/WebsitesTests.ps1 [hunk 2]
```diff
@@ -856,6 +857,13 @@ function Test-SetAzureWebsite
     $website = Get-AzureWebsite $name -Slot Production
     Assert-AreEqual Classic $website.ManagedPipelineMode
     Assert-AreEqual $true $website.WebSocketsEnabled
+
```
Comment (markcowl) [inline src/ServiceManagement/Common/Commands.ScenarioTest/Resources/Websites/WebsitesTests.ps1]: @haocs Didn't this require an update in the test recording?

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/533/

**Comment by hovsepm** [comment]
@haocs you will need to pull from upstream dev. On-Demand job failed due to the issue that is already fixed in dev.

**Comment by haocs** [comment]
@hovsepm on-demand job passed http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/535/

File: src/ServiceManagement/Services/Commands.Utilities/Websites/Services/WebEntities/SiteConfig.cs [hunk 3]
```diff
@@ -63,7 +63,7 @@ public interface ISiteConfig
 
         bool? RemoteDebuggingEnabled { get; set; }
 
-        RemoteDebuggingVersion RemoteDebuggingVersion { get; set; }
```
Comment (naveedaz) [inline src/ServiceManagement/Services/Commands.Utilities/Websites/Services/WebEntities/SiteConfig.cs]: Is the type "RemoteDebugginVersion" used anywhere stil? If not we should delete the type.

**Comment by naveedaz** [comment]
Looks good.

**Comment by akurmi** [comment]
Approved

File: src/ServiceManagement/Services/Commands.Utilities/Websites/Services/WebEntities/SiteConfig.cs (hunk 3 above)
Comment (markcowl) [inline src/ServiceManagement/Services/Commands.Utilities/Websites/Services/WebEntities/SiteConfig.cs]: It has string constants with the known values now, it's not used elsewhere or compilation would have failed.

**Pull request merged**